inc/saiobject.h

```diff
@@ -306,6 +323,24 @@ sai_status_t sai_bulk_object_clear_stats(
         _In_ sai_stats_mode_t mode,
         _Inout_ sai_status_t *object_statuses);
 
+/**
+ * @brief Query the HW stage of an attribute for the specified object type
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_type SAI object type
+ * @param[in] attr_count Count of attributes
+ * @param[in] attr_list List of attributes
+ * @param[out] stage HW stage of the attributes. Length of the array should be attr_count. Caller must allocate the buffer.
```

Comment:
* @param[out] stage HW stage of the object. Caller must allocate the buffer.

Reply:
Query is for a list of attributes and each attribute may belong to different stage. I have enhanced the sample workflow to depict this.

Reply:
It would be cleaner to keep the attr_list identical to a SAI Create API call, i.e., an attribute-id should appear only once in the attr-list. 

This would also allow us to handle the case where the stage of an object depends on more than one attribute-id.